examples/chatgpt/gpt_actions_library/gpt_action_google_calendar.md

```diff
@@ -0,0 +1,359 @@
+
+### Value + Example Business Use Cases
+
+**Value**: Users can now leverage ChatGPT's natural language capability to connect directly to their Google Calendar.
+
+**Example Use Cases**: 
+- You want to create a new event in your calendar. 
+- You want to search your calendar for events based on a specific criteria. 
+- You want to delete an event from your calendar. 
+
+***Note:*** This is a good example of an GPT that may be useful to call from other GPTs using the @<name of your GPT> function. You can find more information on this feature on our [help site](https://help.openai.com/en/articles/8908924-what-is-the-mentions-feature-for-gpts). 
+
+
+## Application Information
+
+### Application Prerequisites
+
+Before you get started, ensure you can meet the following pre-requistes.
+
+ - A Google account with Google Calendar access. 
+ - Permissions to access the Google Calendar API and use the Google Cloud Console to configure your OAuth credentials. 
+
+# Google Calendar Configuration Steps
+
+## Enabling the Google Calendar API
+
+- Visit [console.cloud.google.com](https://console.cloud.google.com).
+- In the project selector, choose the project you’d like to use for this GPT Action. If you don’t have a project yet, click the **Create Project** button.
+- When creating a new project, enter a name for it and select the billing account you’d like to associate. In this example, ‘No Organization’ is selected.
+
+<!--ARCADE EMBED START--><div style="position: relative; padding-bottom: calc(55.43981481481482% + 41px); height: 0; width: 100%;"><iframe src="https://demo.arcade.software/0QxHM3NKyUZcv3di9CkA?embed&embed_mobile=tab&embed_desktop=inline&show_copy_link=true" title="Cookbook | Create Google Cloud Project" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; color-scheme: light;" ></iframe></div><!--ARCADE EMBED END-->
+
+You now have a Google Cloud Project and are ready to configure the API access to your Google Calendar.
+
+- In the Quck Access menu, select **APIs & Services** > **Library**
+- Search for **Google Calendar API** (not DKIM) and click on it.
+- Click on the **Enable** button.
+
+<!--ARCADE EMBED START--><div style="position: relative; padding-bottom: calc(53.793774319066145% + 41px); height: 0; width: 100%;"><iframe src="https://demo.arcade.software/uEOZVBdf8OZ8sP0DZAld?embed&embed_mobile=tab&embed_desktop=inline&show_copy_link=true" title="Cookbook | Enable Google Calendar API" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; color-scheme: light;" ></iframe></div><!--ARCADE EMBED END-->
+
+## Setting up OAuth Credentials
```

Comment:
Would "Creating" be better than "setting"? could be confusing if you're setting it up in chatgpt vs setting in up in GCalendar whereas create suggest GCalendar

Reply:
Yep, will change to creating!